src/state/user/reducer.ts

```diff
@@ -69,6 +70,7 @@ export interface UserState {
   timestamp: number
   audioPlay: boolean
   isDark: boolean
```

Comment:
although it will not be used, keeping this to be ensure having not problems when migrating